src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -11,7 +11,7 @@
 import java.util.Set;
 import java.util.stream.Stream;
 
-import seedu.address.logic.commands.AddCommand;
+import seedu.address.logic.commands.AddSupplierCommand;
```

Comment:
Since AddCommand was renamed to AddSupplier command, should this parser be renamed to AddSupplierCommandParser?